cmd/gossamer/flags.go

```diff
@@ -459,18 +459,19 @@ var (
 	RootFlags = append(GlobalFlags, StartupFlags...)
 
 	// InitFlags are flags that are valid for use with the init subcommand
-	InitFlags = append([]cli.Flag{
+	InitFlags = []cli.Flag{
+		BasePathFlag,
 		ForceFlag,
 		GenesisFlag,
-		PruningFlag,
-		RetainBlockNumberFlag,
-	}, GlobalFlags...)
```

Comment:
Why remove the global flags 🤔 ?

Reply:
the global flags are supposed to be compatible with every subcommand, so they should be readded

Reply:
Why are the global flags supposed to be compatible with every subcommand? None of the global flags apply to the init subcommand.

Reply:
I think all of the global flags apply to the init subcommand? which ones don't apply? log level flags should be usable by every command, same with setting node name, chain, basepath, profiling, and setting pruning settings